managing-os/pentesting.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: doc
+title: Penetration Testing
+permalink: /doc/pentesting/
+---
+
+**Legal notice:**
+
+The usage of penetration testing tools outside your own laboratory environment requires the permission of the organization you attack. Penetration testing without such a permission can have legal consequences.
+
+To avoid such legal conflicts please refer to the [Code of Ethics](https://www.eccouncil.org/Support/code-of-ethics).
+
+Penetration Testing
+===================
+
+"A penetration test, informally pen test, is an attack on a computer system that looks for security weaknesses, potentially gaining access to the computer's features and data." (source [Penetration test](https://en.wikipedia.org/wiki/Penetration_test)).
+
+Penetration Testing Distributions:
+----------------------------------
+
+The following install instructions explain how to setup a penetration testing distribution within Qubes OS.  
+
+- [BlackArch](/doc/blackarch/)
+- [Kali](/doc/kali/)
+- [PenTester Framework (PTF)](/doc/ptf/)
+
+Using Qubes OS to host a "hacking" laboratory
+---------------------------------------------
+
+Qubes OS is a hypervisor based operating system. Qubes OS can various operating systems such as Linux, Unix or Windows in parallel. Qubes OS can therefor be used to host your own "hacking" laboratory.
+
+- [Creating and Using HVM Domains](/doc/hvm-create/)
+- [Templates](/doc/templates/)
```

Comment:
If somebody is new to Qubes OS and penetration hacking, then this person should have a hint. This is the intention behind the second "laboratory" section. 

For people who are familiar with Qubes OS this section does indeed not provide anything valuable. 

In the future the laboratory section could be expanded to explain how to connect to [Hacking Lab](https://www.hacking-lab.com) or how to setup HVM machines such as vulnerable Windows hosts or Metasploitable VM.